873: Update the local tasks as you switch between CR

m4olivei opened the pull request:
Handles the CR side of #873 

This PR makes changes such that when logged in, and you have one or more CR's of content, as you switch between the CR tabs, the local task links will update to reflect the active tab.

## Testing instructions

- [ ] Create content and one or more CR's of that content
- [ ] As an admin, when viewing either the original item, or the CR, click between them using the CR tabs
- [ ] Verify that the local task links (eg. Edit, Delete, etc.) update to reflect the currently active tab

NOTE: Multi-page Items will be handled in a separate pull request.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1249-hpiyxzofydho3t3ynjxmdpsldn5tq4ps.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6939ea385408d4dcd0c13520

m4olivei commented:
> A little vertical space between the local tasks and the community record tabs would be nice (very low priority).

Fixed in recent commit.

> In previous versions a CR could only be created from a base item, not from a CR. I don't actually think it's a permissions issue here, just that seeing the "add community record" button on an existing CR may lead users to think they're doing that, and not adding another to the larger/original item.

Right, so when you click "Add Community Record" while viewing a CR, just like the "+ Community Record" local action before, the node form you're taken to is set to create a new CR with reference to the original content, not the CR. That prevents a user from creating a CR of a CR. I've also taken steps to dynamically show/hide the local task based on if the item in view is a CR or not.

Fixed in recent commit.

> "Create multi-page item" shouldn't show on CRs. If you try to do it, it does give an error message (which is correct), but not displaying it would be better.

Fixed in recent commit.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1249-hpiyxzofydho3t3ynjxmdpsldn5tq4ps.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6939ea385408d4dcd0c13520

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1249-hpiyxzofydho3t3ynjxmdpsldn5tq4ps.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6939ea385408d4dcd0c13520
